core/acl.py

```diff
@@ -113,3 +119,34 @@ async def repeat(self, ctx, *, input: str):
 
     # user's roles are not mapped to any ACL group, return default
     return rule.default
+
+
+def version(major: int, minor: int, micro: Optional[int] = None):
+    """Decorator that specifies minimum discord.py version"""
```

Comment:
`"""Specify minimal discord.py version"""` maybe?

Also, please add short `param` and `return` descriptions, just as `check.acl()` has.